src/input-select/types.ts

```diff
@@ -54,6 +54,7 @@ export interface InputSelectProps<T, V>
     valueToStringFunction?: ((value: V) => string) | undefined;
     /** Function to render selected custom component */
     renderCustomSelectedOption?: ((option: T) => JSX.Element) | undefined;
+    onBlur?: () => void;
```

Comment:
to add the undefined typing per the conventions

```suggestion
    onBlur?: (() => void) | undefined;
```